packages/key-did-resolver/README.md

```diff
@@ -0,0 +1,13 @@
+# did:key method resolver 
+
+> This package contains did:key method resolver implementation
+
+### Installation
+```
+$ npm install @ceramicnetwork/key-did-resolver
+```
+
+## Contributing
+We are happy to accept small and large contributions. Make sure to check out the [Ceramic specifications](https://github.com/ceramicnetwork/specs) for details of how the protocol works.
+
+## License
```

Comment:
Seems to be missing info about the license here?

Reply:
you're right. I have some separate stories regarding the documentation, that should be part of those 👍